src/game/output.c

```diff
@@ -943,6 +943,21 @@ void Output_AnimateTextures(void)
             i--;
             ptr++;
         }
+
+        for (i = 0; i < STATIC_NUMBER_OF; i++) {
+            STATIC_INFO *static_info = &g_StaticObjects[i];
+            if (!static_info->loaded || static_info->nmeshes >= -1) {
+                continue;
+            }
+
+            int32_t num_meshes = -static_info->nmeshes;
```

Comment:
Let's try to apply `const` where possible, also for pointers: `const type *const var = …` to mark that the pointer itself doesn't change

Reply:
Will do 👍

Reply:
Updated. I'd like to rework regular animated textures i.e. potentially parse into structures to make the original part of this function more readable - it's effectively doing the same as the sprite rotations. Another follow-up task I think.